lib/get-seam.ts

```diff
@@ -27,5 +27,12 @@ export const getSeamMultiWorkspace =
     const config = getConfigStore()
     const token = config.get(`${getServer()}.pat`)
     const options = { endpoint: getServer() }
-    return SeamHttpMultiWorkspace.fromPersonalAccessToken(token, options)
+    try {
+      return SeamHttpMultiWorkspace.fromPersonalAccessToken(token, options)
+    } catch (e: any) {
+      if (e.toString().includes("SeamHttpInvalidTokenError")) {
```

Comment:
Note that you can check against the error type directly instead of using the string matching

```ts
import {
  SeamHttp,
  SeamHttpInvalidTokenError,
  SeamHttpMultiWorkspace,
} from "@seamapi/http/connect"

export const getSeamMultiWorkspace =
  async (): Promise<SeamHttpMultiWorkspace> => {
    const config = getConfigStore()
    const token = config.get(`${getServer()}.pat`)
    const options = { endpoint: getServer() }
    try {
      return SeamHttpMultiWorkspace.fromPersonalAccessToken(token, options)
    } catch (e: unknown) {
      if (e instanceof SeamHttpInvalidTokenError) {
        return getSeam() as unknown as SeamHttpMultiWorkspace
      }
      throw e
    }
  }

```